llvm/lib/Transforms/Instrumentation/PGOInstrumentation.cpp

```diff
@@ -319,6 +319,18 @@ static cl::opt<unsigned> PGOFunctionCriticalEdgeThreshold(
     cl::desc("Do not instrument functions with the number of critical edges "
              " greater than this threshold."));
 
+cl::opt<bool> InstrumentColdFunctionCoverage(
+    "instrument-cold-function-coverage", cl::init(false), cl::Hidden,
+    cl::desc("Enable cold function coverage instrumentation (currently only "
+             "used under sampling "
```

Comment:
nit: here and below, the formatting looks odd. Perhaps re-join the doc string and re-run clang-format?